src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBox.cs

```diff
@@ -156,12 +152,8 @@ public async Task When_Calling_Select_With_Out_Of_Range_Length()
 
 			WindowHelper.WindowContent = textBox;
 			await WindowHelper.WaitForLoaded(textBox);
-
-#if __WASM__ // Wasm is behaving differently than UWP and other platforms. https://github.com/unoplatform/uno/issues/7016
-			Assert.AreEqual(10, textBox.SelectionStart);
-#else
-			Assert.AreEqual(0, textBox.SelectionStart);
-#endif
+			textBox.Focus(FocusState.Programmatic);
```

Comment:
Focus shouldn't be needed for the test to pass, I think. But it would be good to verify on UWP.